lib/ex_unit/lib/ex_unit/cli_formatter.ex

```diff
@@ -125,12 +132,56 @@ defmodule ExUnit.CLIFormatter do
     Map.update(test_counter, type, 1, &(&1 + 1))
   end
 
+  ## Slowest
+
+  defp format_slowest_total(%{slowest: slowest} = config, run_us) do
+    slowest_us =
+      config
+      |> extract_slowest_tests()
+      |> Enum.reduce(0, & &1.time + &2)
+
+    slowest_time =
+      slowest_us
+      |> normalize_us()
+      |> format_us()
+
+    percentage = Float.round(((slowest_us / run_us) * 100), 1)
+
+    "Top #{slowest} slowest tests (#{slowest_time}s), #{percentage}% of total time:\n"
```

Comment:
It is probably better to remove the word "tests" from here because it can be properties or anything that integrates with ExUnit.